doc/source/faq.rst

```diff
@@ -1,56 +1,67 @@
+.. index:: FAQ
+.. _FAQ:
+
 FAQ
 ===
 
 **What is iocage?**
-    iocage is jail management script aiming to simplify jail administration
-    tasks as much as possible.
+    :command:`iocage` is a jail management script aiming to simplify
+    jail administration tasks.
 
 **What is a jail?**
-    Jail is a FreeBSD **OS virtualization** technology allowing to run multiple
-    copies of the operating system. Some operating systems use the term
-    **Zones** or **Containers** for OS virtualization.
+    A *Jail* is a FreeBSD **OS virtualization** technology allowing
+    users to run multiple copies of the operating system. Some operating
+    systems use the term **Zones** or **Containers** for OS
+    virtualization.
 
 **What is VNET?**
-    VNET is an independent per jail virtual networking stack.
+    VNET is an independent, per jail virtual networking stack.
 
 **How do I configure network interfaces in a VNET or shared IPjail?**
-    You configure both the same way: ``iocage set
-    ip4_add="interface|IP/netmask" UUID | TAG``. For more info please refer to the
+    Both are configured in the same way:
+    :command:`iocage set ip4_add="[interface]|[IP]/[netmask]" [UUID | TAG]`.
+    For more info, please refer to the :ref:`Networking` section of this
     documentation.
 
 **Do I need to set my default gateway?**
-    Only if VNET is enabled. You need to assign an IP address to the **bridge**
-    where the jail interface is attached to. This IP essentially becomes your default
-    gateway for your jail.
- 
+    Only if VNET is enabled. You need to assign an IP address to the
+    **bridge** where the jail interface is attached. This IP essentially
+    becomes the default gateway for your jail.
+
 **Can I run a firewall inside a jail?**
-    Yes in a VNET jail **IPFW is supported**. PF is not supported inside the
-    jail - though you can still enable PF for the host itself. If you plan on
-    using **IPFW** inside a jail make sure **securelevel** is set to **2**
+    Yes, a VNET jail supports **IPFW**. *PF* is not supported inside the
+    jail. However, you can still enable *PF* for the host. If you plan
+    to use **IPFW** inside a jail, be sure **securelevel** is set to
+    **2**.
 
 **Can I enable both IPFW and PF at the same time?**
     Yes, make sure you allow traffic on both in/out for your jails.
 
 **Can I create custom jail templates?**
-    Yes, and thin provision them too! Starting with version 1.3 there is also a
-    package option for jail packaging.
+    Yes, and thin provisioning is supported too! Starting with version
+    **1.3**, there is also a *package* option for jail packaging.
```

Comment:
Remove Starting *, that's the old version number, and we don't have that feature anymore.